.github/workflows/retry-release.yml

```diff
@@ -0,0 +1,98 @@
+name: Retry Release
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: The Grails Release (e.g. 5.0.3)
+        required: true
+      target_branch:
+        description: The Target Branch (e.g. 5.0.x)
+        required: true
+permissions: {}
+jobs:
+  release:
+    permissions:
+      contents: write  #  to create release
+      issues: read  #  to get closed issues
+
+    runs-on: ubuntu-latest
+    env:
+      GIT_USER_NAME: puneetbehl
```

Comment:
This is fine for now; adding this comment as a reminder that we should consider getting a user/email here for the Grails account/group/project/whatever, rather than using a person account (in this case, Puneet's).